src/main/sensors/gyro.c

```diff
@@ -435,6 +437,14 @@ static uint16_t calculateNyquistAdjustedNotchHz(uint16_t notchHz, uint16_t notch
     return notchHz;
 }
 
+#if !defined(BEEBRAIN)
```

Comment:
Rather than use a target specific #define, instead add at the top of the file, after all the #includes:
```
if !defined(USE_GYRO_SPI_MPU6000) && (FLASH_SIZE > 128)
#define USE_GYRO_INVERSION_FILTER
#endif
```